html-forms/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      HTML Forms
+    </title>
+    <body>
+      <form action="" method="get" class="form example">
+        <h1>Coding Form!</h1>
+
+        <div>
+          <label for="email-address">Email</label>
+          <div>
+            <input type="email" name="Email" id="email-address" />
+          </div>
+        </div>
+
+        <div>
+          <div>
+            <label for="why-coding">Why is coding cool?</label>
+          </div>
+          <div>
+            <select id="why-coding">
+              <option value="">Select an option</option>
+              <option value="interesting">Because it's interesting</option>
+              <option value="fun">Because it's fun</option>
+              <option value="powerful">Because it's powerful</option>
+            </select>
+          </div>
+        </div>
+        <div>
+          <div>
+            <p><label for="favorite-language">Favorite Language(choose one)</label></p>
+          </div>
+          <div>
+            <input type="radio" name="Favorite Language" value="HTML" id="favorite-language" />HTML
```

Comment:
each of the radio inputs needs its own label, you may wrap the inputs with the label if desired